arcgis-ios-sdk-samples/Edit data/Offline edit and sync/OfflineEditingViewController.swift

```diff
@@ -153,32 +153,38 @@ class OfflineEditingViewController: UIViewController {
     }
     
     private func switchToServiceMode() {
+        // Hide the extent view.
+        extentView.isHidden = true
+        // Re-enable interactions.
+        self.mapView.interactionOptions.isPanEnabled = true
+        self.mapView.interactionOptions.isZoomEnabled = true
+        self.mapView.interactionOptions.isRotateEnabled = true
```

Comment:
Consider removing `self` here. Also, what about extracting these 3 lines into a separate method?

Reply:
I considered making a method but it would only be called twice. Would it still be worth separating?